awx/main/dispatch/reaper.py

```diff
@@ -10,6 +10,27 @@
 logger = logging.getLogger('awx.main.dispatch')
 
 
+def startup_reaping():
+    """
+    If this particular instance is starting, then we know that any running jobs are invalid
+    so we will reap those jobs as a special action here
+    """
+    me = Instance.objects.me()
+    jobs = UnifiedJob.objects.filter(status='running', controller_node=me.hostname)
+    job_ids = []
+    for j in jobs:
+        job_ids.append(j.id)
+        j.status = 'failed'
+        j.start_args = ''
+        j.job_explanation += 'Task was marked as running on system startup, so it has been marked as failed.'
```

Comment:
```suggestion
        j.job_explanation += 'Task was marked as running at system start up. The system must have previously not shut down properly, so it has been marked as failed.'
```

Reply:
I might strike out the word "previously" but otherwise I'll take it.